_posts/2020-5-5-updates-improvements-to-pytorch-tutorials.md

```diff
@@ -0,0 +1,72 @@
+
+## PYTORCH RECIPES  
+Recipes are new bite-sized, actional examples designed to teach researchers and developers how to use specific PyTorch features. Some notable new recipes include:
+* [Loading Data in PyTorch](https://pytorch.org/tutorials/recipes/recipes/loading_data_recipe.html)
+* [Model Interpretability Using Captum](https://pytorch.org/tutorials/recipes/recipes/captum_recipe.html)
+* [How to Use TensorBoard](https://pytorch.org/tutorials/recipes/recipes/tensorboard_with_pytorch.html)
+
+View the full recipes (here)[http://pytorch.org/tutorials/recipes/recipes_index.html].
+
+## LEARNING PYTORCH
+This section includes tutorials designed for beginners to PyTorch. Based on community feedback, we have made updates to the current [Deep Learning with PyTorch: A 60 Minute Blitz](https://pytorch.org/tutorials/beginner/deep_learning_60min_blitz.html) tutorial, one of our most popular tutorials for beginners. Upon completion, one can understand what PyTorch and neural networks are, and be able to build and train a simple image classification network. Updates include adding explanations to clarify output meanings and linking back to where users can read more in the docs, cleaning up confusing syntax errors, and reconstructing and explaining new concepts for easier readability.  
+
+## DEPLOYING MODELS IN PRODUCTION
+This section includes tutorials for developers looking to take their PyTorch models to production. The tutorials include:
+* [Deploying PyTorch in Python via a REST API with Flask](https://pytorch.org/tutorials/intermediate/flask_rest_api_tutorial.html)
+* [Introduction to TorchScript](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html)
+* [Loading a TorchScript Model in C++](https://pytorch.org/tutorials/advanced/cpp_export.html)
+* [Exploring a Model from PyTorch to ONNX and Running it using ONNX Runtime](https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html)
+
+## FRONTEND APIS
+PyTorch provides a number of frontend API features that can help developers to code, debug, and validate more efficiently. This section includes tutorials that teach what these features are and how to use them. Some tutorials to highlight: 
+* [Introduction to Named Tensors in PyTorch](https://pytorch.org/tutorials/intermediate/named_tensor_tutorial.html)
+* [Using the PyTorch C++ Frontend](https://pytorch.org/tutorials/advanced/cpp_frontend.html)
+* [Extending TorchScript with Custom C++ Operators](https://pytorch.org/tutorials/advanced/torch_script_custom_ops.html)
+* [Extending TorchScript with Custom C++ Classes](https://pytorch.org/tutorials/advanced/torch_script_custom_classes.html)
+* [Autograd in C++ Frontend](https://pytorch.org/tutorials/advanced/cpp_autograd.html)
+
+## MODEL OPTIMIZATION
+Deep learning models often consume large amounts of memory, battery, and compute due to their complex tasks. This section provides tutorials for model optimization:
```

Comment:
change to: Deep learning models often consume large amounts of memory, power, and compute due to their complexity. This section provides tutorials for model optimization